rex-release/out.sh

```diff
@@ -47,6 +47,20 @@ upload-release() {
 
   # books files with explicit content type set because they don't have extensions, loaded unversioned so no cloudfront caching
   aws s3 sync --exclude 'service-worker.js' --content-type 'text/html' --cache-control 'max-age=0' "$path/books/" "s3://$bucket/rex/releases/$version/books"
+
+  # configure redirects
+  for row in $(cat $path/rex/redirects.json | jq -r '.[] | @base64'); do
```

Comment:
is this base64 to allow you to iterate over each object? you can add the -c flag to remove the whitespace and its a little more straightforward

Reply:
Yeah. `-c` works fine. :)